corehq/apps/user_importer/helpers.py

```diff
@@ -304,3 +325,23 @@ def get_user_primary_location_name(user, domain):
     primary_location = user.get_sql_location(domain)
     if primary_location:
         return primary_location.name
+
+
+def find_differences_in_list(list_to_compare: list, reference_list: list):
```

Comment:
nit: how about `source` and `target` as the names?

Reply:
nit: also seems like a useful generic method that can go in a more generic utils file than under user_importer

Reply:
@mkangia Yeah that could work. (I spent an unusual amount of time thinking of what I'd call that, but even then I was not satisfied)